src/components/Toast/Toast.css

```diff
@@ -0,0 +1,59 @@
+:root{
+    --modal-bg:rgba(0,0,0,0.5);
+}
+
+/* Toast Css */
```

Comment:
A suggestion is to shift this variable to your main components.css, this way it makes it easier to keep track or variables in one file.

Reply:
sure I will do that.